frontend/src/components/maps/leaflet/InfoSlideOut/HeaderActions.tsx

```diff
@@ -0,0 +1,122 @@
+  width: 341px;
+  margin-left: -10px;
+  font-size: 14px;
+  font-family: ProximaNova, sans-serif;
+  padding: 10px;
+  a {
+    padding: 0 10px;
+    color: #1a5a96;
+  }
+`;
+
+const VerticalBar = styled.div`
+  border-left: 6px solid rgba(96, 96, 96, 0.2);
+  height: 18px;
+  margin-left: 1%;
+  margin-right: 1%;
+  border-width: 2px;
+`;
+
+export type IHeaderActions = {
+  /** The selected property */
+  propertyInfo: IParcel | IBuilding | null;
+  /** the selected property type */
+  propertyTypeId: PropertyTypes | null;
+  /** additional action to be taken when a link in the menu is clicked */
+  onLinkClick?: () => void;
+};
+
+/**
+ * Actions that can be done on the property info slide out
+ * @param propertyInfo the selected property information
+ * @param propertyTypeId the selected property type
+ * @param onLinkClick additional action on menu item click
+ */
+const HeaderActions: React.FC<IHeaderActions> = ({ propertyInfo, propertyTypeId, onLinkClick }) => {
+  const leaflet = useLeaflet();
+  const location = useLocation();
+  const keycloak = useKeycloakWrapper();
+  const curZoom = leaflet.map?.getZoom();
+  const jumpToView = () =>
+    leaflet.map?.setView(
+      [propertyInfo?.latitude as number, propertyInfo?.longitude as number],
+      MAX_ZOOM,
+    );
+  const zoomToView = () =>
+    leaflet.map?.flyTo(
+      [propertyInfo?.latitude as number, propertyInfo?.longitude as number],
+      MAX_ZOOM,
+    );
+
+  const buildingId = propertyTypeId === PropertyTypes.BUILDING ? propertyInfo?.id : undefined;
+  const parcelId = propertyTypeId === PropertyTypes.PARCEL ? propertyInfo?.id : undefined;
+  return (
+    <LinkMenu>
+      Actions:
+      <Link
+        onClick={() => {
+          if (curZoom! < MAX_ZOOM) jumpToView();
+        }}
+        to={{
+          pathname: `/mapview`,
+          search: queryString.stringify({
+            ...queryString.parse(location.search),
+            sidebar: true,
+            disabled: true,
+            loadDraft: false,
+            buildingId: buildingId,
+            parcelId: parcelId,
+          }),
+        }}
+      >
+        View
+      </Link>
+      <VerticalBar />
+      {(keycloak.hasAgency(propertyInfo?.agencyId as number) ||
+        keycloak.hasClaim(Claims.ADMIN_PROPERTIES)) && (
+        <>
+          <Link
+            onClick={() => {
+              if (curZoom! < MAX_ZOOM) jumpToView();
+            }}
+            to={{
+              pathname: `/mapview`,
+              search: queryString.stringify({
+                ...queryString.parse(location.search),
+                disabled: false,
+                sidebar: true,
+                loadDraft: false,
+                buildingId: buildingId,
+                parcelId: parcelId,
+              }),
+            }}
+          >
+            Update
+          </Link>
+          <VerticalBar />
+        </>
+      )}
+      {curZoom! < MAX_ZOOM && (
+        <Link to={{ ...location }} onClick={zoomToView}>
```

Comment:
Same thought, what if I open your detail view, zoom in all the way, and then navigate the map away from my property, being able to jump back might be nice.